server.py

```diff
@@ -7,13 +7,18 @@ def main():
     serv.listen(5)
     while True:
         conn, addr = serv.accept()
-        print 'Client from address: {}'.format(addr)
+        print 'Client from address: {} conn: {}'.format(addr, conn)
         data = "Start"
         while data != "":
             data = conn.recv(4096)
-            print data
-            conn.send(data + data)
+            if data == "SHUTDOWN":
```

Comment:
What's going to happen if the client shuts down but doesn't send the SHUTDOWN message? Please handle that case as well

Reply:
fixed